python/pyspark/sql/functions.py

```diff
@@ -1839,15 +1839,19 @@ def from_json(col, schema, options={}):
     string.
 
     :param col: string column in json format
-    :param schema: a StructType or ArrayType of StructType to use when parsing the json column
+    :param schema: a StructType or ArrayType of StructType to use when parsing the json column.
     :param options: options to control parsing. accepts the same options as the json datasource
 
+    .. note:: Since Spark 2.3, the DDL-formatted string is also supported for ``schema``.
```

Comment:
Just to be sure, I think here we allow JSON format schema too but I wonder if we should describe this behaviour to promote this usage as we now have a DDL-formatted string. Actually, one of the reason why DDL was introduced is, inconvenience of this json format schema.